### PR TITLE
Update SOLVERS.md

### PR DESCRIPTION
**Pull request by mateusjatenee**

### CODE REVIEW

**Comment by ryukinix** [comment]
Approved.

**Comment by tkovs** [comment]
Fix your link and to commit --amend.

**Comment by rafaelcn** [comment]
Github review system has improved, looks really nice :-)

File: SOLVERS.md [hunk 1]
```diff
@@ -4,10 +4,12 @@
 ![[lubien](www.github.com/lubien)](https://projecteuler.net/profile/lubien.png)
 ![[leorog](www.github.com/leorog)](https://projecteuler.net/profile/leorog.png)
 ![[tkovs](www.github.com/tkovs)](https://projecteuler.net/profile/tkovs.png)
+![[mateusjatenee](www.github.com/mateusjatenee)](https://projecteuler.net/profile/mateusjatenee.png)
```
Comment (ryukinix) [inline SOLVERS.md]: Fixed.

**Comment by ryukinix** [comment]
@rafaelcn are you ok with that?

**Comment by rafaelcn** [comment]
Shall we merge? Ok, merging.

**Pull request merged**